### PR TITLE
Add rh-postgresql12 role and to katello_devel

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: ansible.cfg [hunk 1]
```diff
@@ -2,7 +2,6 @@
 callback_whitelist = profile_tasks,profile_roles
 stdout_callback = yaml
 display_skipped_hosts = false
-verbosity = 1
```
Comment (rahulbajaj0509) [inline ansible.cfg]: not sure why we need to remove this one though.

File: roles/postgresql_scl/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+- name: 'Install SCL repository'
```
Comment (rahulbajaj0509) [inline roles/postgresql_scl/tasks/main.yml]: should we add a `---` in the beginning  of the playbook?

**Comment by rahulbajaj0509** [comment]
@ehelms in the above PR, we are installing the packages, do we need to also enable their services? so that next time if system reboots or starts then the service would be on.

Let me know your thoughts.

**Comment by ehelms** [comment]
The installer should handle the actual PG service. This is just to configure the environment to be int he context of the SCL. This is updated

**Pull request merged**